x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -492,6 +523,11 @@ export class EndpointDocGenerator {
    * @param options - Allows event field values to be specified
    */
   public generateEvent(options: EventOptions = {}): EndpointEvent {
+    let ancestry: string[] | undefined;
+    if (options?.useAncestryArray === true || options?.ancestry !== undefined) {
```

Comment:
why do we need a `useAncestryArray` option separate from the `ancestry` option?

Reply:
Yeah good point. I'll remove it. We can accomplish the same thing with `ancestry = 0` 👍